### DISCUSSION TITLE
Creating a CSS File?

### DISCUSSION THREAD

**Discussion by bartellramsey** [ExperienceLovelace/ha-floorplan#62] [Q&A] [answered]
By asking this question, I may be showing that I'm wayyyyyyyyyyyy out of my league, but can anyone point me towards some help creating a CSS file?    

Using Inkscape I made an svg file, and I understand how to connect items and the OLD yaml version, but I'm clueless how to even begin a css file.  Is there any information someone can point me towards?

Thanks!

**Comment by exetico** [comment, marked as answer]
The CSS-file are just a normal txt-file, saved as CSS. Take a look at our examples, and select the 'CSS'-tab.

https://experiencelovelace.github.io/ha-floorplan/docs/example-home/ 

You'll need some basic CSS understanding, in order to get most out of ha-floorplan. But there's always a starting-point for everyone 😊👍! Good luck.

**Reply by exetico** [reply]
Any progress?

**Comment by bartellramsey** [comment]
Thank you for your information and your follow up.  I've been making a
crack at it; however, I just got my Home Assistant Blue in the mail, and
have been devoting all my free time to transferring everything over to it!
Very exciting!

I found your example helpful!  I have 0 experience with CSS, so I have a
feeling I'm going to be looking at a lot of examples in the near future.

Thank you!
DB
